packages/wonder-blocks-color/util/utils.js

```diff
@@ -0,0 +1,99 @@
+// @flow
```

Comment:
With Github it's really hard to see any changes that have been made to moved.  Can you create separate commits for moving files and for making changes to them?